Sectors/Lines - External FIR.txt

```diff
@@ -30,6 +30,20 @@ COORD:N055.00.00.000:E005.00.00.000
 COORD:N052.32.24.000:E004.51.12.000
 COORD:N051.23.00.000:E002.00.00.000
 
+SECTORLINE:EDYY_D-J
+COORD:N054.06.20.000:E004.10.48.000
+COORD:N053.19.13.000:E004.43.48.000
+COORD:N052.58.27.000:E005.08.18.000
+
+SECTORLINE:EDYY_J
+COORD:N052.58.27.000:E005.08.18.000
+COORD:N055.00.00.000:E006.30.00.000
```

Comment:
Could be missing a few coords here to align with the ARTCC/High lines, looks like it in yor screenshot as well.
Ref: ARTCC/High/EGTT London FIR.txt

Reply:
With your changes in #2998 in the sector file and an additional line, we have now have this:

![image](https://user-images.githubusercontent.com/12017284/100603720-6b1b5480-32fd-11eb-9a99-42a151b0e566.png)